index.html

```diff
@@ -6,10 +6,86 @@
   </head>
   <body>
     <nav class="navbar">
-      <img class="wolox-logo" src="assets/wolox_logo.svg" alt="Wolox">
+      <img class="wolox-logo" src="assets/wolox_logo.svg">
       <h1 class="title-big">Developer Tools</h1>
     </nav>
-    <footer>
-    </footer>
+    <div class="main-section">
+      <div class="technology-group">
+        <h2 class="title-medium">Front-End Tools</h2>
+
+
+        <div class="technology-list">
+          <div class="technology-card">
+            <img class="tech-logo" src="assets/react_icon.svg">
+            <div class="tech-title">ReactJS</div>
```

Comment:
Change `<div>` for `<h3>` or `<h4>`. It's more descriptive for SEO. Change this in all titles.